R/makeTreeSummarizedExperimentFromqiime2.R

```diff
@@ -159,21 +179,19 @@ read_q2biom <- function(file) {
 
 #' Read qiime2 taxa file
 #' @keywords internal
+#' @importFrom utils read.table
 #' @noRd
-read_q2taxa <- function(file) {
+.read_q2taxa <- function(file) {
     taxa <- utils::read.table(file, sep = '\t', header = TRUE)
-    if ("Confidence" %in% names(taxa)) {
-        taxa$Confidence <- NULL
-    }
-
     as.matrix(taxa)
```

Comment:
The confidence value is `numeric`, but saved currently as `character`. Can you convert this?